charts/alfresco-ooi-service/templates/ingress-ooi-service.yaml

```diff
@@ -0,0 +1,38 @@
+{{- $serviceName :=  printf "%s-%s" (include "alfresco-ooi-service.fullname" .)  "ooi-service" -}}
+{{- $servicePort := .Values.ooiService.service.externalPort -}}
+apiVersion: {{ template "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ template "alfresco-ooi-service.fullname" . }}-ooi-service
+  labels:
+    {{- include "alfresco-ooi-service.labels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+{{- if .Values.ooiService.extraAnnotations }}
+{{ toYaml .Values.ooiService.extraAnnotations | indent 4 }}
+{{- end }}
```

Comment:
That is miss-leading as annotation at this level may not be ingress only. Move it under ingress and amend